var/spack/repos/builtin/packages/r/package.py

```diff
@@ -81,6 +82,7 @@ class R(AutotoolsPackage):
     depends_on('libxt', when='+X')
     depends_on('libxmu', when='+X')
     depends_on('curl')
+    depends_on('pcre2', when='@4:')
     depends_on('pcre')
```

Comment:
Is `pcre` still needed for 4.0 in addition to pcre2?

Reply:
It can be if `pcre>=8.32` and pcre2 won't be available, but I already sent changes that R-4 only depends on `pcre2`